app/operations/workflows/unretire_subjects.rb

```diff
@@ -0,0 +1,23 @@
+# frozen_string_literal: true
+
+module Workflows
+  class UnretireSubjects < Operation
+    validates :workflow_id, presence: true
```

Comment:
not blocking, considering this won't be a widely used end point and if used it's most likely used by our cli / client tools.

We can ensure this is a integer by using active model validations,
- https://github.com/AaronLasseigne/active_interaction#validations
- https://apidock.com/rails/v4.2.9/ActiveModel/Validations/HelperMethods/validates_numericality_of

if you add this it might be worth adding a spec for this on the operation
Something like this might work..
```suggestion
    validates :workflow_id, presence: true, numericality: { greater_than: 0, only_integer: true }
```

if you adopt this you may want to do something like this for the subject_ids as well.